testsuite/tests/libratest/smoke_test.rs

```diff
@@ -1035,14 +1042,18 @@ fn test_e2e_reconfiguration() {
         vec![(10.0, "Coin1".to_string())],
         client_proxy_0.get_balances(&["b", "0"]).unwrap(),
     ));
-    let peer_id = env
-        .get_validator(0)
-        .unwrap()
-        .validator_peer_id()
-        .unwrap()
-        .to_string();
+    let peer_id = env.get_validator(0).unwrap().validator_peer_id().unwrap();
+    let op_tool = OperationalTool::new(
+        format!(
+            "http://127.0.0.1:{}",
+            env.validator_swarm.get_client_port(1)
+        ),
+        ChainId::test(),
+    );
```

Comment:
Nit: Can probably make this a function?  I'm assuming we'll use it in more tests.

Also TIL that you can do `env.validator_swarm.get_client_port(#)`. I had been reading the configs to get that info in my test.